components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -237,3 +237,43 @@ Selecting a project from the list opens the projects's detail page, displaying t
 From there, you can select any individual rule to view its list of conditions, then select a condition to view or update its details.
 
 ![](/images/docs/admin-projects.png)
+
+## Projects in the API
```

Comment:
curious if folks think this section should stay in the overview or be moved to the user guide?

Reply:
I really debated about moving this section, given its use of examples, which is similar to the User Guide. There didn't seem a place to put it in User Guide, and knowing that there was API filtering via CLI would be good to know in an overview sense. Am happy to revisit this content/ placement in the future.